Updated OpenSearch Latest Version (#4768)
platformsh/platformsh-docs: sites/platform/src/add-services/opensearch.md

</table>
<!-- vale off -->

On Grid, from version 2, you only specify the major version. The latest compatible minor version and patches are applied automatically. On Grid, version 1 represents a rolling release - the latest minor version available from the upstream (starting with opensearch 1.3).

> You can see the latest minor and patch versions of OpenSearch available from the [`2.x`](https://docs.opensearch.org/lines/2x.html) and [`1.x`](https://docs.opensearch.org/lines/1x.html) release lines.

Author: ```suggestion
You can see the latest versions of OpenSearch available [here](https://github.com/opensearch-project/OpenSearch/releases).
```

| `analysis-smartcn`      | Smart Chinese Analysis Plugins                                                            | *   | *  | *   |
| `analysis-stempel`      | Stempel Polish Analysis Plugin                                                            | *   | *  | *   |
| `analysis-ukrainian`    | Ukrainian language support                                                                | *   | *  | *   |
| `ingest-attachment`     | Extract file attachments in common formats (such as PPT, XLS, and PDF)                    | *   | *  | *   |
| `mapper-annotated-text` | Adds support for text fields with markup used to inject annotation tokens into the index  | *   | *  | *   |
| `mapper-murmur3`        | Murmur3 mapper plugin for computing hashes at index-time                                  | *   | *  | *   |
| `mapper-size`           | Size mapper plugin, enables the `_size` meta field                                        | *   | *  | *   |
| `repository-s3`         | Support for using S3 as a repository for Snapshot/Restore                                 | *   | *  | *   |

### Alternative plugins

> These plugins are currently available for `OpenSearch 2.0` and `3.0`. The names below show exactly how they should be added to your configuration. For example, to use the `alerting` plugin, specify it in your `services.yaml` as so:

Author: ```suggestion
These plugins are currently available for OpenSearch `2.0` and `3.0`. The names below show exactly how they should be added to your configuration. For example, to use the `alerting` plugin, specify it in your `services.yaml` as so:
```
